sw/device/silicon_creator/lib/drivers/pinmux_unittest.cc

```diff
@@ -0,0 +1,69 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/silicon_creator/lib/drivers/pinmux.h"
+
+#include "gtest/gtest.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/silicon_creator/lib/base/mock_abs_mmio.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+#include "pinmux_regs.h"  // Generated.
+
+namespace pinmux_unittest {
+namespace {
+using testing::Each;
```

Comment:
Nit: leading `::` on `using` statements. (I'm pretty sure we're consistent about this elsewhere.)

Reply:
Ended up just deleting them. Sent a PR (https://github.com/lowRISC/opentitan/pull/7019) to remove other unused `using` statements in the silicon creator driver tests.